### ISSUE TITLE
Add -version option to Xvnc

### ISSUE THREAD

**Issue by durack1** [TigerVNC/tigervnc#163]
I've been trying to ascertain the current version of the installed Xvnc package. If I issue:
```
> Xvnc -version
```
I get an error, whereas the information that I want is included in the error response:
```
Xvnc TigerVNC 1.4.2 - built Jan 23 2015 19:04:52
Copyright (C) 1999-2015 TigerVNC Team and many others (see README.txt)
See http://www.tigervnc.org for information on TigerVNC.              
Underlying X server release 11202000, The X.Org Foundation
```
It'd be useful to add the `-version` option to a future release

**Comment by CendioOssman** [comment]
Sounds reasonable enough. The standard `Xorg` binary has it, so we should as well.

**Comment by CendioOssman** [comment]
Fixed in 20065de.
